src/app/dashboard/revalidation-notes/revalidation-notes.component.scss

```diff
@@ -0,0 +1,32 @@
+.spacer {
```

Comment:
is this spacer class something generic or specific to this component? if its specific to this component probs be good to use BEM for naming the classes to avoid any overlap

Reply:
no-overlap on encapsulated classes is possible. ideally what we'd want is a css structure review i.e. if we actually need individual component sass